stdlib/pervasives.gr

```diff
@@ -48,7 +48,7 @@ from String use { toString, print, concat as (++) }
  * Computes the logical NOT (`!`) of the given operand.
  * Inverts the given Boolean value.
  *
- * @param value: The operand
+ * @param 0: The operand
```

Comment:
Are these positional because we don't have labels on primitives? I believe @spotandjake is going to update all the primitives to have labels like he did in #1979

Reply:
Yep, at this point there aren't labels on primitives

Reply:
#1981